packages/component-library/stories/Box.stories.tsx

```diff
@@ -8,8 +8,11 @@ import {
   Stories,
   PRIMARY_STORY,
 } from "@storybook/addon-docs/blocks"
+import { Paragraph } from "@kaizen/component-library"
 import { Box } from "../components/Box"
 
+import styles from "./Box.stories.scss"
+
 export default {
```

Comment:
Something to consider to move all the docs to the "docs" tab: https://storybook.js.org/docs/react/writing-docs/docs-page#remixing-docspage-using-doc-blocks

If these are left in the stories themselves we would start running visual regressions on them 

```
export default {
  title: 'Box (React)',
  component: Box,
  parameters: {
    docs: {
      page: () => (
        <>
          <Title />
          <Subtitle />
          <Description />
          <Primary />
          <Documentation /> // add this
          <ArgsTable story={PRIMARY_STORY} />
          <Stories />
        </>
      ),
    },
  },
};
```